src/App.js

```diff
@@ -8,16 +8,19 @@ import BottomNavBar from "./common/BottomNavBar";
 import Home from "./home";
 import Navigation from './navigation';
 import Account from "./account";
+import { useAuthentication } from "./common/useAuthentication";
 
 function App() {
+  const {user} = useAuthentication();
```

Comment:
I did not understand this line (please add the necessary comments). You did not use the user object here.

Reply:
Thank you. I think I see the issue with the Firebase authentication.